features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -1138,7 +1138,7 @@
                 <DisplayOrder>3</DisplayOrder>
 				<Required />
 				<SupportedByDefault />
-                <RegEx>^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$</RegEx>
+                <RegEx>^\s*(?:\+?(\d{1,3}))?[-. (]*(\d{3})[-. )]*(\d{3})[-. ]*(\d{4})(?: *x(\d+))?\s*$</RegEx>
```

Comment:
Looks like there's some indenting mismatch. Shall we fix this

Reply:
The indentation issue does not exist in the original file.

Reply:
ack